fighters/link/src/acmd/aerials.rs

```diff
@@ -154,35 +116,29 @@ unsafe fn attack_air_b(fighter: &mut L2CAgentBase) {
         WorkModule::on_flag(boma, *FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING);
     }
     frame(lua_state, 6.0);
+    FT_MOTION_RATE(fighter, 1.25);
```

Comment:
lets change this to the standard `FT_MOTION_RATE` format `FT_MOTION_RATE(fighter, C / (B - A))`